test/liblangutil/CharStream.cpp

```diff
@@ -28,6 +28,25 @@
 
 #include <boost/test/unit_test.hpp>
 
+using namespace std;
+
+namespace std
+{
+template<typename T>
+std::ostream& operator<<(std::ostream& _ostream, std::optional<T> const& _value)
```

Comment:
I think you've done this because Boost.Test makes use of it for failure outputs?

Reply:
I think that adding stuff to `std` is undefined behavior though, which is why I've usually been doing it this way instead:
https://github.com/ethereum/solidity/blob/a7d89a92434cea438f3b212fe6360a8d2a304011/test/solc/CommandLineInterface.cpp#L50-L108

Reply:
Also, it would be nice to have this definition in `test/Common.h` or some file it includes so that you don't have to define it every time locally.

Reply:
anonymous namespace is not visible across compilation units though. I think `ostream& operator<<(ostream&, T)` should simply be in the same namespace as `T` to be found by ADL (argument dependent lookup), isn't it?